hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/ScmConfigKeys.java

```diff
@@ -372,6 +372,13 @@
       "ozone.scm.pipeline.per.metadata.disk";
 
   public static final int OZONE_SCM_PIPELINE_PER_METADATA_VOLUME_DEFAULT = 2;
+
+  public static final String OZONE_DATANODE_METADATA_VOLUME_FREE_SPACE_MIN =
+      "ozone.scm.datanode.metadata.volume.free-space.min";
+
+  public static final String
+      OZONE_DATANODE_METADATA_VOLUME_FREE_SPACE_MIN_DEFAULT = "0MB";
```

Comment:
Suggest set a different default value other than 0.

Reply:
Hmmm,  "0MB" means we just keep the old behavior as default, if we are using some other values, then we should have some rationale ?

Reply:
Because we want to make sure ratis log dir will not be fully occupied, right?

Reply:
Yes, I'll choose a reasonable default for ozone.

Reply:
I think here we could reserve the space of a pipeline with raft logs at max(purge.gap default to 1 million) and 1KB should be enough for a log entry, then we have 1GB space in total. In our ozone deployments, the biggest pipeline is about 100MB at most, so 1GB shall be quite enough and 1GB should not be a waste for modern disks.